packer/install-dependencies.sh

```diff
@@ -364,10 +364,10 @@ K8S_VERSIONS="
 1.13.6
 1.13.5
 1.13.5-azs
+1.12.9
+1.12.9-azs
```

Comment:
@rjaini FYI

Reply:
Thx for adding.

Reply:
@CecileRobertMichon : Can you please update AzureStack (https://github.com/Azure/aks-engine/blob/master/docs/topics/azure-stack.md) doc as well to include 1.12.9.

Reply:
already did https://github.com/Azure/aks-engine/pull/1428/files#diff-c14705e88bd9472ced61a1de53fe1a20R157